packages/core/database/lib/metadata/index.js

```diff
@@ -21,7 +21,13 @@ const createMetadata = (models = []) => {
   const metadata = new Metadata();
 
   // init pass
+  const seenTables = new Map();
   for (const model of _.cloneDeep(models)) {
+    if (seenTables.get(model.tableName)) {
```

Comment:
@petersg83 - this check in the database layer that errors out when it detects clashing DB names.

Would you suggest I either revert the CTB changes or create the new route for checking DB table names?

Reply:
Cool :) It will fix the error if 2 content-types have the same table names. However there are other tables (relational tables) that are automatically created and that can clash also. I believe that the best place to do the check is after the metadata has been created. I believe in the constructor in `database/lib/index.js`.

Regarding the CTB, let's discuss about it during our meeting later?